herbert_core/admin/herbert_version.m

```diff
@@ -1,4 +1,4 @@
-function varargout = herbert_version()
+function varargout = herbert_version(varargin)
```

Comment:
may be worth adding non-empty varargin examples to the usage section. But I don't see any usage of varargin - why are you providing it?

Reply:
new version of `herbert_version` accepts arguments. As it may be unclear which version is used during initialization or autocheck for version may be performed I have added the argument to old version to maintan the same interface.
Do not see need for providing usage for the interface yet but may be necessary in a future